ads/README.md

```diff
@@ -39,6 +39,7 @@ We will provide the following information to the ad:
 - `window.context.pageViewId` contains a relatively low entropy id that is the same for all ads shown on a page.
 - [ad viewability](#ad-viewability)
 - `window.context.startTime` contains the time at which processing of the amp-ad element started.
+- `window.context.data.container` contains the name of ad container element if exist.
```

Comment:
@lannka I just realized this when I update the doc.
Is `window.context.data.container` a good place to put the container name? All other info is stored in `window.context`, data only contains info where user defined?